### DISCUSSION TITLE
Could you please provide guidance on how we can become a sponsor?

### DISCUSSION THREAD

**Discussion by IamYJLee** [AliveToolkit/alive2#1032] [General]
@nunoplopes 

Dear nuno,

We've been utilizing alive2 to some extent and are interested in exploring sponsorship opportunities.
Could you please provide guidance on how we can become a sponsor?

Best regards,
Jun

**Comment by nunoplopes** [comment]
Thank you for your interest and nice to hear that you've been using Alive2!

Right now we process sponsorships as donations to our research lab (https://www.inesc-id.pt). INESC-ID is a private, non-profit institution that conducts R&D. Most of its members are professors at the university of Lisbon (myself included).
Due to the overheads involved in receiving sponsorships, we are only accepting donations of €50k+ (or equivalent).

If you are interested, please drop me an [email](mailto:nuno.lopes@tecnico.ulisboa.pt) and I'll get the process started.

Thanks!
